### ISSUE TITLE
Add WASM Target?

### ISSUE THREAD

**Issue by hendursaga** [klemola/vetovoima#5]
It would be great if one could compile this to WebAssembly - that way, you could feature it on your Itch.io page without someone having to download an executable.

**Comment by klemola** [comment]
@hendursaga I agree! It's on the roadmap. I haven't had the time to even try it yet, but maybe I can do it for the next release, which could come out in the next couple of months.
